src/controlers/users.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const userServices = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userServices.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+
+  if (!userId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = userServices.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundUser);
```

Comment:
Do it in all cases
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/91411692/3e8fdca6-609d-45ce-a360-60baf4ba9103)